.gitignore

```diff
@@ -212,4 +212,8 @@ src/vscode-autorest/**/*.js
 src/client/**/*
 
 src/extension/old/**/*
-*.d.ts
\ No newline at end of file
+*.d.ts
+
```

Comment:
Yup, I do not want to see these files on git status and i believe they are not either checked-in :)